server.py

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#                                                                              #
+# This file implements the http server and the handler of http requests        #
+#                                                                              #
+################################################################################
+
+# http.server and socketserver are imported to implement the http server
+import http.server
+import socketserver
+# extenso imported to write down the number
+import extenso
+
+# this is the custom handler that 
+class Handler(http.server.BaseHTTPRequestHandler):
+    # the only thing that needs to be modified is create the do_GET method 
+    def do_GET(self):
+        self.send_response(200)
+        self.send_header("Content-type", "json")
+        self.end_headers()
+        print(self.path)
+        try:
+            # converting the path to a workable number
+            num = int(self.path[1:])
+            # creation of the json formated string using the "extenso" function
+            json = "{\"extenso\": \""+ extenso.extenso(num) +"\"}"
+            self.wfile.write(json.encode())
+        except:
+            # if the path isn't a number, an error should be displayed
+            json = "{\"extenso\": \"O PATH NAO E UM NUMERO\"}"

```

Comment:
nitpick: geralmente é melhor mandar num campo separado, p.ex. "error"

Reply:
Verdade, creio que não prestei atenção nisso na hora de limpar o código para enviá-lo.

```suggestion
            json = "{\"error\": \"O PATH NAO E UM NUMERO\"}"
```